README.md

```diff
@@ -62,9 +62,24 @@ For help, please consider the following venues (in order):
 * [File an issue](https://github.com/aws/amazon-vpc-cni-k8s/issues/new/choose)
 * Chat with us on the `#aws-vpc-cni` channel in the [Kubernetes Slack](https://kubernetes.slack.com/) community.
 
+## Recommended Version
+
+For all Kubernetes versions, we recommend installing the latest VPC CNI release. The following table denotes our minimum recommended
+VPC CNI version for each actively supported Kubernetes version.
+
+| Kubernetes Version |    1.25    |    1.24    |    1.23    |    1.22    |    1.21    |    1.20    |    1.19    |
```

Comment:
Do we need to add 1.21 and below in this table since that version is no longer supported?

Reply:
I thought about that, but since we still have customers using it, I figured it did not hurt to add

Reply:
Should we remove `actively supported`  as it might be confusing ?

Reply:
Good point, maybe I should just follow https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

Reply:
Updated the table in latest revision